src/app/table/view/header/header-cell.component.html

```diff
@@ -5,7 +5,7 @@
         <div (click)="onSortClick(columnInfo)" [class.clickable]="columnInfo.sortable" id="sort">
           <ng-template *ngIf="columnInfo.contentTemplateRef" [ngTemplateOutlet]="columnInfo.contentTemplateRef">
           </ng-template>
-          <span *ngIf="columnInfo.caption">{{ columnInfo.caption }}</span>
+          <span style="word-break: break-word" *ngIf="columnInfo.caption">{{ columnInfo.caption }}</span>
```

Comment:
Minor nitpick - put the `ngIf` before the `style` attribute - it's more important not to miss it when reading the template